app/views/pages/developer-guide/data-import-export/import-files.liquid

```diff
@@ -7,7 +7,7 @@ metadata:
 
 This topic describes importing files: **images** and **attachments**.
 
-Models and Users can store images and attachments in properties. (Visit [User Uploads](/developer-guide/user-uploads/user-uploads)) To import them you need to have the proper data model in [*.yml files](/developer-guide/properties). Here is an example model with images and attachments:
```

Comment:
@diana-lakatos please note, @piotrze has recently added a new way of seeding the data which use property `upload` and which we use in our template - we might want to add it to the documentation

Reply:
@Slashek Thank you! I opened a ticket for this task: #1340